rest-api-spec/src/main/resources/rest-api-spec/test/search/30_limits.yml

```diff
@@ -116,9 +116,6 @@ setup:
 
 ---
 "Regexp length limit":
-  - skip:
-      version: all
-      reason: Long regex breaks HTTP query length when request body is sent as a query param - https://github.com/elastic/elasticsearch/issues/65718
```

Comment:
NB on backport this needs to restore a previous skip version that looks like this:
```
 - skip:
      version: " - 6.99.99"
      reason: "The regex length limit was introduced in 7.0.0"

```

Reply:
Thanks!